src/assets/scripts/client/InputController.js

```diff
@@ -842,7 +988,11 @@ export default class InputController {
      * @private
      */
     _onRightMousePress(event) {
-        this._markMousePressed(event, MOUSE_BUTTON_NAMES.RIGHT);
+        if (this.input.measureTool.isMeasuring) {
```

Comment:
```js
if (!this.input.measureTool.isMeasuring) {
    this._markMousePressed(event, MOUSE_BUTTON_NAMES.RIGHT);

    return;
}

this._removeLastMeasurePoint();
```

Reply:
I know it's much of a muchness, but would it be better to put the `_removeLastMeasurePoint` call inside the if? That way if more tests are ever added, the default behaviour will be the original `_markMousePressed`